test/appium/tests/test_dapps_and_browsing.py

```diff
@@ -21,3 +21,5 @@ def test_browse_link_entering_url_in_dapp_view(self):
         browsing_view.back_to_home_button.click()
 
         assert home_view.first_chat_element_title.text == 'Status | The Mobile Ethereum Client'
+        # TODO: enable when https://github.com/status-im/status-react/issues/3013 is closed
+        # home_view.swipe_and_delete_chat('Status | The Mobile Ethereum Client')
```

Comment:
https://github.com/status-im/status-react/issues/3013 is closed, please uncomment the line